partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,156 @@
+        "update",
+    ]
+
+    if ctx.invoked_subcommand in commands_with_context:
+        from partcad.globals import init
+
+        try:
+            if p is None:
+                ctx.obj = init()
+            else:
+                ctx.obj = init(p)
+        except (yaml.parser.ParserError, yaml.scanner.ScannerError) as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)